static/coffee/services.coffee

```diff
@@ -549,3 +549,14 @@ services.factory('UtilsService', ['$window', '$uibModal', ($window, $uibModal) -
       return $uibModal.open({templateUrl: '/partials/modal_daterange.html', controller: 'DateRangeModalController', resolve: resolve}).result
 
 ])
+
+
+#=====================================================================
+# Pod API service
+#=====================================================================
+services.factory('PodApi', ['$window', '$http', ($window, $http) ->
```

Comment:
Sorry I didn't notice this before but can we call this `PodService` like the other services ?